integrationtest/backend/tck/src/main/java/org/hibernate/search/integrationtest/backend/tck/search/highlight/AbstractHighlighterIT.java

```diff
@@ -426,6 +426,23 @@ void numberOfFragmentsSingle() {
 				);
 	}
 
+	@Test
+	void numberOfFragmentsSingleNamedHighlighter() {
+		StubMappingScope scope = index.createScope();
+
+		SearchQuery<String> highlights = scope.query().select(
+				f -> f.highlight( "anotherString" ).single().highlighter( "single" )
```

Comment:
While updating the processor I've noticed that the previous commit didn't address the case when there's a single fragment + a named highlighter, so I moved highlighter step methods around 🙈